projects/cm_mcu/CommandLineTask.c

```diff
@@ -341,6 +367,7 @@ static struct command_t commands[] = {
     },
     {"psmon", psmon_ctl, "Displays a table showing the state of power supplies.\r\n", 1},
     {"psreg", psmon_reg, "<which> <reg>. which: LGA80D (10*dev+page), reg: reg address in hex\r\n", 2},
+    {"reset_all", reset_all, "Reset all LGA80Ds on their two pages all at once\r\n", 0},
```

Comment:
Maybe change help message to ‘reset all LGA80D snapshot registers’

Reply:
And also change the command name to `sn_reset` or something less generic

Reply:
Done.